bip-0327.mediawiki

```diff
@@ -606,7 +606,7 @@ Algorithm ''DeterministicSign(sk, aggothernonce, pk<sub>1..u</sub>, tweak<sub>1.
 ** The secret signing key ''sk'': a 32-byte array
 ** The aggregate public nonce ''aggothernonce'' (see [[#modifications-to-nonce-generation|above]]): a 66-byte array
 ** The number ''u'' of individual public keys with ''0 < u < 2^32''
-** The individual public keys ''pk<sub>1..u</sub>'': ''u'' 32-byte arrays
```

Comment:
Given that the line right above states that keys are between 0 < u < 2^32, it seems to me that we are looking at an x-only key and the text is already correct, but maybe I’m misinterpreting that.

Reply:
> maybe I’m misinterpreting that.

You are. `u` is the number of public keys involved in signing and not related to their length.